src/routes/Sensors/DataTable/index.tsx

```diff
@@ -53,30 +64,54 @@ function Row({ data }: RowProps) {
           </IconButton>
         </TableCell>
         <TableCell component="th" scope="row">
-          {data.timestamp}
+          <Typography whiteSpace="nowrap">{data.timestamp}</Typography>
+        </TableCell>
+        <TableCell align="right">
+          <Typography
+            whiteSpace="nowrap"
+            maxWidth="12rem"
+            overflow="hidden"
+            textOverflow="ellipsis"
+          >
+            {data.nodeId}
+          </Typography>
         </TableCell>
-        <TableCell align="right">{data.sensorPosition}</TableCell>
```

Comment:
Ah sorry there was a misunderstanding about position, I was thinking about latitude and longitude, not the field sensorPosition for now. Can we have a "position field" with lat/lon?

Reply:
I believe it becomes too crowded if we add the location also there. I also believe that the decoded value will possibly not fit nicely and we may have to reconsider the column selection. Let's leave them like this for now and return once we have more things to show.